### PR TITLE
LTR 9 cards

### PR DESCRIPTION
**Pull request by Simisays**
so it begins ....

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/galadriel,gift_giver.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Galadriel, Gift-Giver
+ManaCost:3 G G
+Types:Legendary Creature Elf Noble
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCharm| TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, ABILITY
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigCharm| TriggerZones$ Battlefield | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, ABILITY
+SVar:TrigCharm:DB$ Charm | Choices$ DBCounter,DBTreasure,DBFood
+SVar:DBCounter:DB$ PutCounter | CounterType$ P1P1 | ValidTgts$ Creature | TgtPrompt$ Select target creature | CounterNum$ 1 | SpellDescription$ Put a +1/+1 counter on target creature.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/galadriel,gift_giver.txt:8]: Wrong ValidTgts

File: forge-gui/res/cardsfolder/upcoming/storm_of_saruman.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Storm of Saruman
+ManaCost:4 U U
+Types:Enchantment
+K:Ward:3
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | Execute$ TrigCopyTarget | CheckSVar$ YouCastThisTurn | SVarCompare$ EQ2 | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast your second spell each turn, copy it, except the copy isn't legendary. You may choose new targets for the copy. (A copy of a permanent spell becomes a token.)
+SVar:TrigCopyTarget:DB$ CopyPermanent | Defined$ TriggeredCard | NonLegendary$ True | Choices$ Card
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/storm_of_saruman.txt:6]: wrong effect

File: forge-gui/res/cardsfolder/upcoming/galadriel_gift_giver.txt [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-SVar:DBTreasure:DB$ Token | TokenScript$ c_a_treasure_sac | SpellDescription$ Create a Treasure token. (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
-SVar:DBFood:DB$ Token | TokenScript$ c_a_food_sac | SpellDescription$ Create a Food token. (It's an artifact with "{2}, {T}, Sacrifice this artifact: You gain 3 life.")
-SVar:HasAttackEffect:TRUE
-DeckHas:Ability$Token|Counters|LifeGain|Sacrifice & Type$Food|Treasure
+Name:Galadriel, Gift-Giver
+ManaCost:3 G G
+Types:Legendary Creature Elf Noble
+PT:4/4
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCharm| TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, ABILITY
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigCharm| TriggerZones$ Battlefield | Secondary$ True | TriggerDescription$ Whenever CARDNAME enters the battlefield or attacks, ABILITY
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/galadriel_gift_giver.txt:6]: Space after both TrigCharm

File: forge-gui/res/cardsfolder/upcoming/nasty_end.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Nasty end
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/nasty_end.txt:1]: Nasty End

File: forge-gui/res/cardsfolder/upcoming/storm_of_saruman.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Storm of Saruman
+ManaCost:4 U U
+Types:Enchantment
+K:Ward:3
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | Execute$ TrigCopyTarget | CheckSVar$ YouCastThisTurn | SVarCompare$ EQ2 | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast your second spell each turn, copy it, except the copy isn't legendary. You may choose new targets for the copy. (A copy of a permanent spell becomes a token.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/storm_of_saruman.txt:5]: Use `ActivatorThisTurnCast$`

**Review by tool4ever — APPROVED** [review]
thanks

**Pull request merged**